### PR TITLE
.spawn instead of .exec to avoid buffering output

### PR DESCRIPTION
**Pull request by psuter**
In the NodeJS container, buffering the full output causes issues with archives that contain many files.

### CODE REVIEW

**Comment by psuter** [comment]
PG1 751 is... colorful

File: core/nodejsActionBase/runner.js [hunk 1]
```diff
@@ -159,13 +161,28 @@ function NodeActionRunner(whisk) {
...
+
+                child.stdout.on("data", function (data) {
+                    lastOut = data;
+                });
+
+                child.stderr.on("data", function (data) {
+                    lastErr = data;
+                });
+
+                child.on("close", function (code) {
```
Comment (rabbah) [inline core/nodejsActionBase/runner.js]: handle on `error` also? https://nodejs.org/api/child_process.html#child_process_event_error

**Comment by rabbah** [comment]
@psuter retry travis or chase down the failure?

**Comment by psuter** [comment]
Superseeded by #1540

**Pull request closed without merging**